### PR TITLE
Add form validation, using form component

### PR DESCRIPTION
**Pull request by sukhrajghuman**

### CODE REVIEW

File: docs/js/validate.js [hunk 1]
```diff
@@ -0,0 +1,215 @@
+const fullName             = document.getElementById( 'name' );
+const email                = document.getElementById( 'email' );
+const helpDescription      = document.getElementById( 'help-comment' );
+const form                 = document.querySelector( 'form' );
+const errorSummary         = document.getElementById('error-summary');
```
Comment (adamzerella) [inline docs/js/validate.js]: spacing here

File: docs/js/validate.js [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+const errorSummaryMessages = document.querySelector( '#error-summary p' );
+const errorMessageHeading  = document.querySelector( '#error-summary h3' );
+const errors               = { name: '' , email: '' , 'help-comment': ''};
+
+/**
+ * Add event listener when form submitted
+ */
+form.addEventListener( 'submit', function( event ) {
+	if ( validateForm() ) {
```
Comment (adamzerella) [inline docs/js/validate.js]: What's the point of this?

File: docs/js/validate.js [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+	}
+	else {
+		PreventEvent( event );
+		generateErrorSummary();
+		RemoveClass( errorSummary, 'hide_content' );
+		errorMessageHeading.focus();
+	}
+});
+
+// Validation
```
Comment (adamzerella) [inline docs/js/validate.js]: Double comments

File: package.json [hunk 4]
```diff
@@ -60,6 +60,7 @@
 		"@gov.au/cta-link": "latest",
 		"@gov.au/direction-links": "latest",
 		"@gov.au/footer": "latest",
+		"@gov.au/form": "^0.1.0",
```
Comment (adamzerella) [inline package.json]:
```suggestion
		"@gov.au/form": "latest",
```

File: docs/js/validate.js (hunk 2 above)
Comment (sukhrajghuman) [reply docs/js/validate.js]: woops forgot to get rid of it after a refactor

**Pull request merged**